.pre-commit-config.yaml

```diff
@@ -0,0 +1,55 @@
+ci:
+  autofix_prs: false
+
+repos:
+  # Lint first (with safe autofixes)
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.6.9
+    hooks:
+      - id: ruff
+        name: ruff-check
+        args: [--fix, --show-fixes]
+        files: ^(kubeflow-pipelines/(docling-standard|docling-vlm)/.*\.py|[^/]+\.py)$
+
+  # Fast formatter (optional alongside Black; keeps layout consistent)
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.6.9
+    hooks:
+      - id: ruff-format
+        name: ruff-format
+        files: ^(kubeflow-pipelines/(docling-standard|docling-vlm)/.*\.py|[^/]+\.py)$
+
+  # Import ordering (aligned with Black)
+  - repo: https://github.com/pycqa/isort
+    rev: 5.13.2
```

Comment:
Is there a reason you're using 5.13.2 instead of 6.0.1?

Reply:
I'll bump this